### DISCUSSION TITLE
Typescript errors in Sandpack

### DISCUSSION THREAD

**Discussion by andreifloricel** [codesandbox/sandpack#155] [Q&A] [answered]
Hi, 
is there a way to get the typescript (compilation) errors in sandpack?
Currently (ex. with template 'react-ts') I don't get the errors in the Sandpack editor but I get them if I open/fork the files in Codesandbox.

**Comment by DeMoorJasper** [comment]
Compilation errors should show, typing errors will not

**Comment by danilowoz** [comment, marked as answer]
This conversation is still happening [here](https://github.com/codesandbox/sandpack/discussions/237)
